### PR TITLE
 fix(EditableTextfield): allow empty value for EditableTextfield

### PR DESCRIPTION
**Pull request by robstax**
Resolves: https://github.com/collab-ui/collab-ui-react/issues/129

### CODE REVIEW

File: src/lib/Input/index.js [hunk 1]
```diff
@@ -212,7 +212,7 @@ class Input extends React.Component {
         onFocus={this.handleFocus}
         onMouseDown={this.handleMouseDown}
         placeholder={placeholder}
-        value={value || defaultValue}
+        value={typeof value === 'string' ? value : defaultValue}
```
Comment (robstax) [inline src/lib/Input/index.js]: issue was `value=""` and `""` is falsy

Comment (bfbiggs) [reply src/lib/Input/index.js]: This negates defaultValue completely. I would suggest:
```
  state = {
    isEditing: false,
    value: this.props.value || this.props.defaultValue,
  };
```

And make this value={value}

Comment (bfbiggs) [reply src/lib/Input/index.js]: defaultValue would also need to be omitted from ...props.

File: src/lib/EditableTextfield/index.js [hunk 2]
```diff
@@ -106,6 +106,7 @@ class EditableTextfield extends React.PureComponent {
             defaultValue={inputText}
```
Comment (bfbiggs) [inline src/lib/EditableTextfield/index.js]: Appears it would make sense to remove defaultValue?

Comment (robstax) [reply src/lib/EditableTextfield/index.js]: I was considering that. I was wondering if it was supposed to be a fallback if you didn't pass the `prop` at all.

Comment (bfbiggs) [reply src/lib/EditableTextfield/index.js]: Since the value is static now, I would remove it.

**Pull request merged**